src/universal/mutations/ApproveToOrgMutation.ts

```diff
@@ -1,14 +1,14 @@
-import {commitMutation} from 'react-relay'
-import {showInfo} from 'universal/modules/toast/ducks/toastDuck'
+import {commitMutation, graphql} from 'react-relay'
 import ClearNotificationMutation from 'universal/mutations/ClearNotificationMutation'
 import getNotificationsConn from 'universal/mutations/connections/getNotificationsConn'
 import handleAddInvitations from 'universal/mutations/handlers/handleAddInvitations'
 import handleAddNotifications from 'universal/mutations/handlers/handleAddNotifications'
 import handleRemoveNotifications from 'universal/mutations/handlers/handleRemoveNotifications'
 import handleRemoveOrgApprovals from 'universal/mutations/handlers/handleRemoveOrgApprovals'
-import popTeamInviteNotificationToast from 'universal/mutations/toasts/popTeamInviteNotificationToast'
 import filterNodesInConn from 'universal/utils/relay/filterNodesInConn'
 import getInProxy from 'universal/utils/relay/getInProxy'
+import {ApproveToOrgMutation_notification} from '__generated__/ApproveToOrgMutation_notification.graphql'
```

Comment:
Are the generated files just a bunch of typescript?

Reply:
the relay compiler climbs through every file looking for graphql tagged template literals & then climbs down each referenced fragment like a tree, which is why they have such strict naming conventions (FileName_varName). the end result is generated file with 3 parts:
- a function describing the entire query (including all the fragments it traversed)
- a typescript definition of the returned object
- a checksum so it can throw a warning if the generated file is stale

the definitions get their nullability from the graphql schema, so i get friendly little warnings telling me some fields may be coming back null from the server.